operator/src/main/java/oracle/kubernetes/operator/wlsconfig/WlsServerConfig.java

```diff
@@ -250,6 +255,10 @@ public void setSslListenPort(Integer listenPort) {
     this.sslListenPort = listenPort;
   }
 
+  public PortDetails getSslListenPortDetails() {
```

Comment:
Is this method added for future use, it doesn't seem to be used anywhere?

Reply:
I think I used it at one point, but no longer do, so I'll delete it. Thanks for catching this.

Reply:
It looks like you deleted the other method but not this one. Is this intentional?

Reply:
I thought I'd gotten them all, but I only deleted two of the three. I'm trying to get this merged, and afraid if I keep making changes it keeps requiring more reviews. @rjeberhard should we defer deleting this?